CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ cmake_policy(SET CMP0048 NEW)
 # patch = bump on any change
 # (tweak is not used by libpredict)
 project(libpredict VERSION 3.0.0 LANGUAGES C CXX)
+add_definitions(-D_POSIC_C_SOURCE=1)
```

Comment:
Is "POSIC" correct?

Reply:
Nope. :-)

When I fixed that, M_PI disappeared. I've decided on _XOPEN_SOURCE=700 which should enable both gmtime_r and M_PI.